tracee-ebpf/tracee/tracee.go

```diff
@@ -173,6 +177,11 @@ func (tc Config) Validate() error {
 	return nil
 }
 
+type profilerInfo struct {
+	Times    int64  `json:"times,omitempty"`
```

Comment:
I'm considering to remove `Times`. Sometimes, Tracee startup is slower than expected and as a result we miss on events which cause noise.
<img width="673" alt="image" src="https://user-images.githubusercontent.com/1254783/119740992-b5f3d000-be39-11eb-9741-3fd84e63483e.png">


I think it's not very useful to know how many times a program was executed. Let me know if you think otherwise.

Reply:
New output:
```
 cat /tmp/tracee/out/tracee.profile
{
  "4026531840:/proc/1/root/snap/microk8s/2213/sbin/xtables-multi:1620854501000000000": {
    "file_hash": "474a0724ecba3b22a267810f2b0c10c792d3f2090464e2d614b59959491e59b9"
  },
  "4026531840:/proc/1/root/usr/sbin/sshd:1615393641978322322": {
    "file_hash": "9169f72f64ff6e0cc7597449b90312889275c613b4fcec390a852683cdd085d0"
  }
}
```

Reply:
I believe we said we will wait for tracee to start (tracee.pid created) before moving on

Reply:
OK I tested this approach and it seems to work. I will add the times back as discussed offline.